contributors/devel/coding-conventions.md

```diff
@@ -75,7 +75,7 @@ following Go conventions - `stateLock`, `mapLock` etc.
 tests
 
   - Table-driven tests are preferred for testing multiple scenarios/inputs; for
-example, see [TestNamespaceAuthorization](../../test/integration/auth/auth_test.go)
+example, see [TestNamespaceAuthorization](https://github.com/kubernetes/kubernetes/blob/master/test/integration/auth/auth_test.go)
```

Comment:
Can we use "https://git.k8s.io/kubernetes/test/integration/auth/auth_test.go" for this?